src/swap/SwapExecuteScreen.tsx

```diff
@@ -147,6 +158,10 @@ export function SwapExecuteScreen() {
 
   return (
     <SafeAreaView style={styles.safeAreaView}>
+      <CustomHeader
```

Comment:
do we need a CustomerHeader here? couldn't it be a View that has 100% width with some padding and justifyContent flex-end, and the TopBarIconButton?

Reply:
I think it makes sense to keep the custom headers using the same component as it will be easier to manage changes to headers in the future. Looking at #3004 changes having the logic for custom headers spread out throughout components makes it harder to change across the app.

Reply:
hmm i agree only partly - if this were a screen header then for sure we should be using the header component. but this seems just a cross icon to me (like you'd see on a modal, since it doesn't have the traditional components you'd expect on a header, like a back button or informative text) so it feels strange to use a header component where we need to "hack" it by passing an empty View as the "leftComponent" :/

but i defer to your judgement, if you feel strongly that we should use a custom header component to render the cross icon then we can leave it, it's not such a big deal :)

Reply:
The passing of an empty item is a bit strange; however, if I remember correctly this is the only screen in the app that has only a button on the right and the only screen with a back button on the right. @nityarao should we consider moving this close button to the left for consistency across the app?